opencode/skills/AGENTS.md

```
@@ -0,0 +1,19 @@
+# SKILLS KNOWLEDGE BASE
+
+## OVERVIEW
+Agent skill definitions and architectural patterns.
+Skills define specialized capabilities injected into agents via `load_skills`.
+
+## STRUCTURE
+```
+skills/
+├── agent-skill-architect/ # Complex skill with multiple files
+└── config-modernizer.md   # Single-file skill definition
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)